cmd/metrics.go

```diff
@@ -35,16 +35,33 @@ func isValidMetric(metric string) bool {
 	return false
 }
 
-func getMetricDependenciesDirect(adjacencyList AdjacencyList) MetricDependenciesDirectType {
-	depsCount := make(MetricDependenciesDirectType)
+/*
+Get direct dependencies given adjacency list (used both for dependencies and dependents)
```

Comment:
Comments [should start ](https://go.dev/wiki/CodeReviewComments#comment-sentences) with a function name.